src/current/_includes/cockroachcloud/first-org-user-roles.md

```diff
@@ -0,0 +1,3 @@
+{{site.data.alerts.callout_info}}
+The user who creates a new organization is assigned a combination of Org Administrator, Billing Coordinator, and Cluster Admin at the organization scope. Any of these roles may subsequently be removed, although another user must have the Org Adminstrator role before it can be removed.
```

Comment:
The removal statement also applies to Cluster Admin, so would have to weave that in there.